src/marqo/core/models/score_modifier.py

```diff
@@ -0,0 +1,13 @@
+from enum import Enum
+from marqo.base_model import StrictBaseModel
+
+
+class ScoreModifierType(Enum):
+    Multiply = 'multiply'
+    Add = 'add'
+
+
+class ScoreModifier(StrictBaseModel):
+    field: str
```

Comment:
Did you move these from somewhere? Pretty sure we had these before

Reply:
Are you referring to ```class ScoreModifierOperator(BaseModel):
    field_name: str
    weight: float = 1```
defined in the `score_modifier_object.py` file?

Reply:
Yes I moved these, they used to be in the search query file. I thought it was better they have their own module. Becuase I'm also using them in different places now